test_getval.c

```diff
@@ -0,0 +1,18 @@
+#include "shell.h"
```

Comment:
The source code displays path, so edit and remove this main.c test file so i can merge